src/codeflare_sdk/cluster/config.py

```diff
@@ -48,3 +48,4 @@ class ClusterConfiguration:
     instascale: bool = False
     envs: dict = field(default_factory=dict)
     image: str = "ghcr.io/foundation-model-stack/base:ray2.1.0-py38-gpu-pytorch1.12.0cu116-20221213-193103"
+    pull_secret: dict = field(default_factory=dict)
```

Comment:
Just to be sure, do we want to support multiple pull secrets, or one is fine?

Reply:
Hi @astefanutti,
I've made the necessary adjustments to the pr allow for multiple image pull secrets. 

Also in doing so, was able to remove the alias prevention method.

Reply:
Sorry for the delayed response. Yes, supporting multiple secrets sounds good to me.